### DISCUSSION TITLE
Can I run this on an existing RPI 4

### DISCUSSION THREAD

**Discussion by rlust** [spbrogan/rvc2mqtt#125] [General]
Can I run this docker container on an existing RPI4 that is currently running HASSIO current release?

**Comment by spbrogan** [comment]
If you know what you are doing, I expect so.  You still need the host os to enable can support.  You also need to then map that hardware into the docker container.   I was looking into making it a home assistant "add-on" but never finished that work.   In the end I decided that I would prefer independent hardware because they aren't in the same physical location.   Keep me posted if you do explore this.

**Comment by WatersPaul** [comment]
Dd you get this to work on a PRPI4?
